CRM/Emailfiling/Service/MailProcessor.php

```diff
@@ -1,13 +1,11 @@
 <?php
 
-namespace Civi\Emailfiling;
-
 /**
  * Class MailProcessor
  *
  * @package Civi\EmailFiling
  */
-class MailProcessor {
+class CRM_Emailfiling_Service_MailProcessor {
```

Comment:
I asked if we can add tests for this class?

Reply:
I don't think I can write them myself, so either we can do it together or leave for the next stage of development.